src/ape/api/accounts.py

```diff
@@ -95,8 +95,15 @@ def call(self, txn: TransactionAPI, send_everything: bool = False) -> ReceiptAPI
         if send_everything:
             if txn.max_fee is None:
                 raise TransactionError(message="Max fee must not be None.")
-
-            txn.value = self.balance - txn.max_fee
+            if not txn.gas_limit or txn.gas_limit is None:
+                raise TransactionError(message="The txn.gas_limit is not set.")
+            else:
+                txn.value = self.balance - (txn.max_fee * txn.gas_limit)
+            if txn.value <= 0:
+                raise ValueError(
+                    f"Sender does not have enough to cover transaction value and gas: \
```

Comment:
It would be nice to have a test hit this error!

Reply:
Hmm maybe trying to send from an empty account would do it

Reply:
yes that works.